packages/react-dom/src/client/__tests__/setInnerHTML-test.js

```diff
@@ -1,68 +0,0 @@
-/**
- * Copyright (c) 2016-present, Facebook, Inc.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- *
- * @emails react-core
- */
-
-'use strict';
-
-// TODO: can we express this test with only public API?
-var setInnerHTML = require('../setInnerHTML').default;
-var Namespaces = require('../../shared/DOMNamespaces').Namespaces;
-
-describe('setInnerHTML', () => {
-  describe('when the node has innerHTML property', () => {
-    it('sets innerHTML on it', () => {
-      var node = document.createElement('div');
-      var html = '<h1>hello</h1>';
-      setInnerHTML(node, html);
-      expect(node.innerHTML).toBe(html);
-    });
-  });
-
-  describe('when the node does not have an innerHTML property', () => {
-    var node;
-    var nodeProxy;
-    beforeEach(() => {
-      // Create a mock node that looks like an SVG in IE (without innerHTML)
-      node = document.createElementNS(Namespaces.svg, 'svg');
-
-      nodeProxy = new Proxy(node, {
```

Comment:
Removing this causes the test to no longer verify the SVG IE code path. You can test it yourself by commenting out the related parts in `setInnerHTML`: the test passes despite them being broken.